data/mods/Aftershock/items/ammo/7.50mm.json

```diff
@@ -14,7 +14,27 @@
     "stack_size": 47,
     "ammo_type": "afs_7.50mm",
     "range": 36,
-    "damage": { "damage_type": "bullet", "amount": 44, "armor_penetration": 2 },
+    "damage": { "damage_type": "bullet", "amount": 45, "armor_penetration": 15 },
+    "dispersion": 30,
+    "recoil": 1500,
+    "extend": { "effects": [ "COOKOFF" ], "flags": [ "CASELESS_ROUNDS" ] }
+  },
+  {
+    "id": "afs_7.50mm_caseless",
```

Comment:
afs_7.50mm_caseless (itype) has two definitions from the same source (aftershock)!

Reply:
What I get for thinking I could change the math of my own mod with only the json styler as an assurance I didnt mess up.